operations/observability/mixins/meta/rules/components/server/alerts.libsonnet

```
@@ -150,8 +134,9 @@
           },
           {
             alert: 'WebAppServicesCrashlooping',
-            expr: 'sum(rate(container_cpu_usage_seconds_total{container!="POD", node=~".*", pod=~"(content-service|dashboard|db|db-sync|messagebus|payment-endpoint|proxy|server|ws-manager-bridge|usage)-.*"}[5m])) by (pod, node) > 0.80',
-            'for': '15m',
+            // Reasoning: alert if any pod is restarting more than 3 times / 5 minutes.
+            expr: 'rate(kube_pod_container_status_restarts_total{container!="POD", node=~".*", pod=~"(content-service|dashboard|db|db-sync|messagebus|payment-endpoint|proxy|server|ws-manager-bridge|usage)-.*"}[15m]) * 60 * 3 > 0',
```

```suggestion
            expr: 'increase(kube_pod_container_status_restarts_total{container!="POD", node=~".*", pod=~"(content-service|dashboard|db|db-sync|messagebus|payment-endpoint|proxy|server|ws-manager-bridge|usage)-.*"}[5m]) > 3,
```

If you just want to measure raw increase in 5m, the `increase()` can help simplify the query 🙂

---

The filter `node=~".*"` could also be removed, right? It's not actually filtering anything, just making the query heavier to the backend 😬

---

Done.